MUXSDKStats/MUXSDKStats/MUXSDKStats.h

```diff
@@ -257,7 +257,7 @@ FOUNDATION_EXPORT
 
  */
 + (void)videoChangeForPlayer:(nonnull NSString *)name
-            withCustomerData:(nullable MUXSDKCustomerData *)customerData;
+            withCustomerData:(nonnull MUXSDKCustomerData *)customerData;
```

Comment:
This is a breaking change and requires a major version bump. Particularly for Swift callers.

I'm not sure if it's absolutely required too implement the required functionality in this PR, though. Based on everything else here I don't feel like it is?

Reply:
Good point. I don't need to make this nullable.